content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Adding a Google Photos Cloud Sync Task"
+description: "This article provides instructions on how to set up Google Photos API credentials and use them to create a cloud sync task."
+weight: 40
+tags:
+- scalecloud
+- scalesgooglephotos
+- scalebackup
+---
+
+{{< toc >}}
+
+Google Photos works best in TrueNAS using a Google Photos API key and [rclone](https://rclone.org/) token.
+
+## Creating the API Credentials
+
+On the [Google API dashboard](https://console.cloud.google.com/apis/dashboard), click the dropdown menu next to the Google Cloud logo and select your project. If you do not have a project, click **NEW PROJECT** and enter a **Project name**, **Organization**, and **Location**.
```

Comment:
small change suggestion. As a style suggestion (from most of the style guides out there), we don't want to use the UI elements in this way so I suggest this:
```suggestion
On the [Google API dashboard](https://console.cloud.google.com/apis/dashboard), click the dropdown menu next to the Google Cloud logo and select your project. If you do not have a project, click **NEW PROJECT** and enter a value in **Project name**, **Organization**, and **Location**.
```